package com.springbook.biz.common;

public class AfterThrowingAdvice {
	public void exceptionLog() {
		System.out.println("[Exception] 로직 수행 중 예외 발생");
	}

}
